modules/apps/data-engine/data-engine-test/src/testFunctional/tests/macros/DataEngineUpgrade.macro

```diff
@@ -65,9 +65,16 @@ definition {
 
 	macro validateNStructure {
 
-		// TODO LPS-127401 validateNStructure pending implementation
-
-		echo("empty stub");
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoStructures();
+       
+        for (var structureTitle : list "My Structure with all fields,My Structure with nested fields,My Structure with fieldsets level 0,My Structure with fieldsets level 1,My Structure with fieldsets level 2") {		
```

Comment:
Hey @DiegoDOC , as discussed before. Next time you can declare the list outside of the loop to increase readability of your code.